### DISCUSSION TITLE
Theory behind Exudyn?

### DISCUSSION THREAD

**Discussion by jgerstmayr** [jgerstmayr/EXUDYN#7] [Theory] [answered]
Thank you for EXUDYN, I have another question, how can I study these theroy in EXUDYN? what book could you recommend for me ? I have finished `Ahmed A. Shabana - Dynamics of Multibody Systems-4th.pdf` and I am reading `Dynamics of Flexible Multibody Systems Rigid Finite Element Method` , but I am still perplexed. There are several methods for `Dynamics of Multibody Systems` .

**Comment by jgerstmayr** [comment, marked as answer]
Several details on the specific implementations are written in docs/theDoc.pdf and there are also a couple of references on multibody systems. Nevertheless, there is no single book to explain the various approaches and methods used in rigid and flexible multibody dynamics systems. There are also summer schools, which can help you to get started.

**Comment by tianhaichen** [comment]
> Thank you for EXUDYN, I have another question, how can I study these theroy in EXUDYN? what book could you recommend for me ? I have finished `Ahmed A. Shabana - Dynamics of Multibody Systems-4th.pdf` and I am reading `Dynamics of Flexible Multibody Systems Rigid Finite Element Method` , but I am still perplexed. There are several methods for `Dynamics of Multibody Systems` .

Do you have the codes about DADS discussed in the book?

**Comment by jgerstmayr** [comment]
No, I do not have the DADS code. But you can find many theories implemented in Exudyn's C++ part.
It depends what you need to know in particular. Details can be only found in scientific papers.
